owncloudApp/src/main/java/com/owncloud/android/workers/CameraUploadsWorker.kt

```diff
@@ -124,10 +124,13 @@ class CameraUploadsWorker(
             else -> SyncType.PICTURE_UPLOADS
         }
 
+        val currentTimestamp = System.currentTimeMillis()
+
         val localPicturesDocumentFiles: List<DocumentFile> = getFilesReadyToUpload(
             syncType = syncType,
             sourcePath = folderBackUpConfiguration.sourcePath,
-            lastSyncTimestamp = folderBackUpConfiguration.lastSyncTimestamp
+            lastSyncTimestamp = folderBackUpConfiguration.lastSyncTimestamp,
+            currentTimestamp = currentTimestamp
```

Comment:
Maybe we should start using [trailing commas](https://kotlinlang.org/docs/coding-conventions.html#trailing-commas) just as @abelgardep suggested 😀

Reply:
👍🏻